testgrid/tgrun/pkg/runner/vmi/embed/runcmd.sh

```diff
@@ -436,21 +442,29 @@ function wait_for_cluster_ready() {
         fi
         echo "cluster is not ready"
         i=$((i+1))
-        if [ $i -gt 10 ]; then
+        if [ $i -gt 20 ]; then
             report_failure "cluster_not_ready"
             exit 0
         fi
-        sleep 120
+        sleep 60
     done
 }
 
+function useHA() {
```

Comment:
@emosbaugh need your suggestion here, when I run the installation script  function with ``ha`` it give me this error
```
kubernetes.loadBalancerAddress required
```
SO from what I read in the doc I have add this to the spec
```
    ekco:
      version: "latest"
      enableInternalLoadBalancer: true  
```
and then I added this function and everything started to work fine but If you have a better way let me know.

Reply:
yes please use `enableInternalLoadBalancer: true`